adapters/handlers/rest/configure_api.go

```diff
@@ -198,13 +201,36 @@ func MakeAppState(ctx context.Context, options *swag.CommandLineOptionsGroup) *s
 	appState := startupRoutine(ctx, options)
 
 	if appState.ServerConfig.Config.Monitoring.Enabled {
-		appState.ServerMetrics = monitoring.NewServerMetrics(appState.ServerConfig.Config.Monitoring, prometheus.DefaultRegisterer)
+		appState.ServerMetrics = monitoring.NewServerMetrics(appState.ServerConfig.Config.Monitoring.MetricsNamespace, prometheus.DefaultRegisterer)
 		appState.TenantActivity = tenantactivity.NewHandler()
 
 		// export build tags to prometheus metric
 		build.SetPrometheusBuildInfo()
 		prometheus.MustRegister(version.NewCollector(build.AppName))
 
+		opts := armonprometheus.PrometheusOpts{
+			Expiration: 0, // never expire any metrics,
+			Registerer: prometheus.DefaultRegisterer,
+		}
+
+		sink, err := armonprometheus.NewPrometheusSinkFrom(opts)
+		if err != nil {
+			appState.Logger.WithField("action", "startup").WithError(err).Fatal("failed to create prometheus sink for raft metrics")
+		}
+
+		cfg := armonmetrics.DefaultConfig("weaviate_internal") // to differentiate it's coming from internal packages.
```

Comment:
I find the namespace names here confusing:
- `weaviate_internal` is for raft metrics.
- `weaviate_raft` is for internal cluster communication via RPC metrics.
Are they backwards or are the names intentional? If intentional, can you think of any names that would be more clear for someone who hasn't read the code?

Perhaps:
- `weaviate_raft` for the raft metrics.
- `weaviate_cluster_rpc` for the internal cluster communication via RPC metrics.

Reply:
Good point.

But note `weaviate_internal` is not just for raft, it can also have memberlist related metrics. My rule of thumb here is, if the metrics is coming from our dependencies, then it all will have `weaviate_internal` prefix.

But I agree, I can convert current `weviate_raft` -> `weviate_cluster` (denoting coming from our weviate cluster communication)

So here are the changes I'm going to do
1. `weaviate_raft` -> `weaviate_cluster`
2. `weaviate_internal` - keeping as it is (in practice, it can have metrics like `weaviate_internal_raft_*` and `weaviate_internal_memberlist_*`, etc.

That way we have clear separation about, the metrics we instrumented and metrics coming from our dependencies.